src/nvim/testdir/runtest.vim

```diff
@@ -161,9 +161,8 @@ func RunTheTest(test)
     endtry
   endif
 
-  " Make "Q" switch to Ex mode.
-  " This does not work for all tests.
-  nnoremap Q gQ
+  " Align Nvim defaults to Vim.
+  source vim_defaults.vim
```

Comment:
naming it with setup_ prefix helps discoverability a bit, then it will live next to setup.vim

Reply:
Hmm, using a `setup_` prefix causes `Test_set_completion()` to fail.

Reply:
I'll just mark the a part of `setup.vim` that is always run.